### PR TITLE
Support copying file in the debugger

### PR DESCRIPTION
**Pull request by grouma**
This comes from an internal request. Add  a simple copy button to copy the file's name. 

Screen Shot 2021-04-08 at 11 38 59 AM

### CODE REVIEW

File: packages/devtools_app/lib/src/debugger/codeview.dart [hunk 1]
```diff
@@ -348,6 +349,11 @@ class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
                   style: theme.textTheme.subtitle2,
                 ),
               ),
+              ToolbarAction(
```
[the remark this answers is not in the document]
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/debugger/codeview.dart:352]: +1

**Review by kenzieschmoll — APPROVED** [review]
lgtm_ship_it

**Comment by jacob314** [comment]
Why are we adding a toolbar icon instead of using SelectableText?

**Comment by kenzieschmoll** [comment]
> Why are we adding a toolbar icon instead of using SelectableText?

Maybe we should do both. I do like the copy to clipboard button, but I also like the idea of having the text be selectable too.

**Comment by grouma** [comment]
> > Why are we adding a toolbar icon instead of using SelectableText?
> 
> Maybe we should do both. I do like the copy to clipboard button, but I also like the idea of having the text be selectable too.

@jacob314 and I discussed offline. I think I'm going to go with a tripple dot icon which has a dropdown copy option. Eventually we'll add a bunch more actions like open in code search etc.

**Comment by grouma** [comment]
I also found the chevron icon confusing so I updated the icon to be history and update the tooltip:

Screen Shot 2021-04-08 at 2 10 12 PM

**Comment by kenzieschmoll** [comment]
Should we get some UX input on changing the caret to the history icon? IntelliJ, for example, uses a caret for their dropdown menu of files:
Screen Shot 2021-04-08 at 4 13 42 PM

at first glance, the history icon looks like a refresh button to me

**Comment by grouma** [comment]
> uses a caret for their dropdown menu of files

 The caret made sense when it was the only dropdown. Sitting next to the triple dot icon makes it not vey informative. I find the history icon more descriptive. Regardless the tooltip text helps a lot.

File: packages/devtools_app/lib/src/debugger/codeview.dart [hunk 2]
```diff
@@ -349,8 +350,36 @@ class _CodeViewState extends State<CodeView> with AutoDisposeMixin {
                 ),
               ),
               const SizedBox(width: denseSpacing),
+              PopupMenuButton<_ScriptPopupMenuOptions>(
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/debugger/codeview.dart:353]: nit: this build method is getting pretty long. can you break these items into two helpers `_buildScriptOptionsPopupMenu` and `_buildScriptHistoryPopupMenu`

**Comment by kenzieschmoll** [comment]
lgtm_army

File: packages/devtools_app/lib/src/debugger/codeview.dart (hunk 2 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/debugger/codeview.dart:353]: better yet, two stateless widgets

**Comment by jacob314** [comment]
lgtm

**Pull request merged**